pkg/client/informers/externalversions/generic.go

```diff
@@ -23,9 +23,8 @@ import (
 
 	v1alpha1 "github.com/rook/rook/pkg/apis/cassandra.rook.io/v1alpha1"
 	v1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
-	v1beta1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1beta1"
 	cockroachdbrookiov1alpha1 "github.com/rook/rook/pkg/apis/cockroachdb.rook.io/v1alpha1"
-	edgefsrookiov1beta1 "github.com/rook/rook/pkg/apis/edgefs.rook.io/v1beta1"
+	v1beta1 "github.com/rook/rook/pkg/apis/edgefs.rook.io/v1beta1"
```

Comment:
It looks like this might be something your editor did automatically? It seems good to me to still identify the backend so that things are clearer in the below code.

On that note, `v1` might be better as `rookv1`, `v1alpha2` as `rookv1alpha2`, and `v1alpha1` as `cassandrav1alpha1`.

Ultimately this is a nit.

Reply:
`make codegen` updated this generated code. Agreed that having the storage provider in the import name is helpful though.